README.md

```diff
@@ -2,22 +2,23 @@
 
 # Project Name
 
-> Portfolio: mobile version.
+> Review one of your past projects
```

Comment:
[Optional] You should have mentioned the description of the project be the original description of the project you are requested to review it yourself in this activity. The `Review one of your past projects` is an instruction to do a review on a previously done project which is assumed to have a description already.